### PR TITLE
Add python 3 instructions, couple other tweaks

### PR DESCRIPTION
**Pull request by bobcatfish**
One fix and a couple of python 3 tweaks!

Fix:
* Added missing indentation for a for loop

Added updates for python 3: the docs say 2.7 is required, but it seems like sometimes folks show up with python 3 (and 2.7 will be deprecated eventually!) so makes sense to add the alternatives for python 3 to the slides:
- `raw_input` vs. `input`
- `next(foo)` vs. `foo.next()` (the latter being discouraged anyway!)

(It might make sense to have a slide about how to know if you're running python 2 or 3 also but I didn't feel confident enough in my french to add a slide to both!)

@christinatruong

### CODE REVIEW

File: slides.html [hunk 1]
```diff
@@ -528,12 +528,20 @@ <h1 class="heading-bg">
 
         Instead of just a 'Hello World' message on the screen, take it one step further and have Python say hello to anyone!
 
-        Try using the `raw_input()` function. This will get the data inputed by a user and **returns** that value back to you.
-
-        1. Give `raw_input()` a message to let the user know what information is being requested using this syntax (note the quotes):
+        The function `raw_input` in Python 2 (or `input` if you have installed Python3) will get the data input by the user and **return* the value back to you.
```
Comment (tiffanytse) [inline slides.html]: Add space on `Python 3` 😄

**Comment by tiffanytse** [comment]
If we can fix that one typo I'll merge :)

File: slides.html (hunk 1 above)
Comment (bobcatfish) [inline slides.html]: Good catch, thanks @tiffanytse !

**Comment by bobcatfish** [comment]
Typo should be fixed @tiffanytse !

**Pull request merged**